lib/migration_helper.rb

```diff
@@ -6,7 +6,7 @@ def rename old_name, new_name
     down_old = old_name.downcase
     down_new = new_name.downcase
     Rails.logger.info "Change name from '#{cap_old}' to '#{cap_new}'"
-    Card[cap_old].update_attributes! :name=>cap_new, :update_referencers=>true, :silent_change=>true
+    Card[cap_old].update_attributes! :name=>cap_new, :update_referers=>true, :silent_change=>true
```

Comment:
Line is too long. [97/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.